db/compaction/compaction_picker.cc

```diff
@@ -1040,6 +1042,13 @@ Status CompactionPicker::SanitizeCompactionInputFilesForAllLevels(
       }
     }
   }
+
+  if (!valid_input_files_found) {
```

Comment:
Does checking is_first works here? Or rename is_first to valid_input_files_found for better naming.